api4/context.go

```diff
@@ -375,6 +375,38 @@ func (c *Context) RequireUsername() *Context {
 	return c
 }
 
+func (c *Context) RequireTeamName() *Context {
+	if c.Err != nil {
+		return c
+	}
+
+	if len(c.Params.TeamName) < 2 {
+		c.SetInvalidUrlParam("team_name")
+	} 
+
+	if len(c.Params.TeamName) > 15 {
+		c.SetInvalidUrlParam("team_name")
+	}
+
+	return c
+}
+
+func (c *Context) RequireChannelName() *Context {
+	if c.Err != nil {
+		return c
+	}
+
+	if len(c.Params.ChannelName) < 2 {
+		c.SetInvalidUrlParam("channel_name")
+	} 
+
+	if len(c.Params.ChannelName) > 22 {
```

Comment:
Can we use constants for these numbers? I think `model.CHANNEL_NAME_MAX_LENGTH` already exists but you'll need to add `model.CHANNEL_NAME_MIN_LENGTH`

Also if you don't mind updating the function `IsValidChannelIdentifier()` in `model/utils.go` to use the new minimum constant

Same for above with the team name